docs/general/builtins/scalar.rst

```diff
@@ -938,6 +938,20 @@ fractional seconds (milliseconds) are given.
     depends on the system clock and the JVM implementation.
 
 
+.. _current_date:
+
+``CURRENT_DATE``
+----------------
+
+The ``CURRENT_DATE`` expression returns the date in UTC timezone at the time the SQL statement was handled. Clock
```

Comment:
Thanks for your contribution!

Would you mind wrapping this to 79 chars per line?